Library/OcCompilerIntrinsicsLib/OcCompilerIntrinsicsLib.c

```diff
@@ -6,10 +6,11 @@
 **/
 
 #include <Library/BaseLib.h>
+#include <Library/BaseMemoryLib.h>
 
 GLOBAL_REMOVE_IF_UNREFERENCED BOOLEAN  gOcCompilerIntrinsicsLib;
 
-#if defined (MDE_CPU_IA32) && defined (__clang__) && defined (__apple_build_version__) && __apple_build_version__ < 11000000
+#if defined (MDE_CPU_IA32) && defined (__clang__) && defined (__APPLE__)
```

Comment:
FWIW I did need to remove this version test, __udivdi3 was getting generated by code in TlsDxe when compiling using Xcode 15.0.1